std-bits/table/src/main/java/org/enso/table/data/column/operation/CountNothing.java

```diff
@@ -0,0 +1,64 @@
+package org.enso.table.data.column.operation;
+
+import org.enso.table.data.column.storage.ColumnStorage;
+import org.enso.table.data.column.storage.ColumnStorageWithNothingMap;
+import org.enso.table.data.table.Column;
+import org.graalvm.polyglot.Context;
+
+/** An operation for counting the number of Nothing values in a Column. */
+public class CountNothing {
+  // ** Counts the number of Nothing values in the given column. */
```

Comment:
```suggestion
  /** Counts the number of Nothing values in the given column. */
```

I think the syntax we try to mostly use for such comments is `/** ... */` - so it is a single-line but still a block-comment and not the `//`.